### ISSUE TITLE
IOS bug when repaint view

### ISSUE THREAD

**Issue by Xiangfs** [boynet/Framework7-Indexed-List-plugin#14]
when i moving the finger through the letters, the view actually scrolls into the letter area, but it shows nothing on screen. Then i use the developer tool of safari to find whether the element is still on the screen or not, and the result is yes, is just something gets wrong when ios system repaint the view. When i trying to scroll the screen, the elements shows up again! 
I just want to know how to deal with this issue, otherwise, everything goes well on android.

**Comment by Xiangfs** [comment]
the following pictures are the situations when i moving my fingers through the letters.

![img_1437](https://user-images.githubusercontent.com/20461849/27279971-e4d48204-5518-11e7-87d7-c546776022f3.PNG)
￼
![img_1438](https://user-images.githubusercontent.com/20461849/27279996-f6982806-5518-11e7-87b6-39006436813f.PNG)
￼
![img_1436](https://user-images.githubusercontent.com/20461849/27280005-fec55166-5518-11e7-8d39-db8328706a4b.PNG)

**Comment by boynet** [comment]
Hi does it also happen on the demo page? https://boynet.github.io/boynet/

**Comment by Xiangfs** [comment]
No, it only happens in real phone, i think something goes wrong with my elements' levels and css style, but i can't find the differents between our css style and elements' level. 
I'll try it on my simulator to see whether it happens or not on simulator and tell you later.

**Comment by boynet** [comment]
Hi any results? I know this kind of bugs is usually gpu memory problem

**Comment by Xiangfs** [comment]
till now, there has no one who goes through with the same issue ? Because i haven't see any issues about the situation that i deal with.
So, if you now the main reason, do you have some solution to resolve this problem?
I tried many methods, but got no one which can fix it.

**Comment by boynet** [comment]
can you upload some demo where this bug happen?

**Comment by Xiangfs** [comment]
It's a pity that our app only works on our wi-fi environment, even if I upload our app, you can't login and see the problem too.

**Comment by FredrikLastow** [comment]
I have the same problem. Did anyone find a solution for this? When I scroll
